app/src/main/java/com/sdp/movemeet/backend/providers/AuthenticationInstanceProvider.java

```diff
@@ -0,0 +1,21 @@
+package com.sdp.movemeet.backend.providers;
```

Comment:
What is the goal of this class ? May be you can add some documentation

Reply:
You're right! I forgot to document it. They're static classes that we can use to retrieve Firebase backend instances across all classes. This is useful for abstraction purposes and testing: if you replace the instance in the static class by a mocked or spied one, then all subsequent calls to `getInstance()` will retrieve this fake instance.